documentation/docs/steps/checkChangeInDevelopment.md

```diff
@@ -6,6 +6,37 @@
 
 * No prerequisites
 
+**Note:** This step is deprecated. Use the [isChangeInDevelopment](isChangeInDevelopment.md) instead.
+
+You can keep most of the step specific configuration parameters in your configuration file `config.yml` untouched. The new step support the old naming convention. However, it is recommended to adjust your parameters to the new steps.
+
+Following parameters are not supported anymore. Adjust as indicated.
+
+| Unsupported Parameter | Change Notice |
```

Comment:
I would write "New Parameter" instead of "Change Notice", and write just the parameter name in the table, without the "Use ... instead. "